SU2_CFD/include/output/filewriter/CParallelDataSorter.hpp

```diff
@@ -232,4 +244,20 @@ class CParallelDataSorter{
   su2double GetUnsorted_Data(unsigned long iPoint, unsigned short iField){
     return connSend[Index[iPoint] + iField];
   }
+  
+  /*!
+   * \brief Get the vector containing the names of the output fields
+   * \return Vector of strings containing the field names
+   */
+  vector<string> GetFieldNames(){
+    return fieldNames;
+  }
```

Comment:
This can probably be returned by const&, it will still allow a local copy to be created if necessary.